### DISCUSSION TITLE
Create a commandline flag to render multiple scenes to one video

### DISCUSSION THREAD

**Discussion by WampyCakes** [ManimCommunity/manim#2782] [Suggestions and Proposals]
## Description of proposed feature
A new commandline flag (what that flag should be called is open to debate) that when used would render the selected scenes to one video file. If `-a` is passed, it should render all of the scenes. If a comma separated list of scene names is passed, only those scenes should be written to the file. If neither are passed, default to all scenes.

## How can the new feature be used?
It removes the need to use/install video editing software or figure out ffmpeg commands for simple cases.

## Additional comments
I think this should be achievable using ffmpeg by just joining the rendered scenes together into one video. Transitions between the scenes should be handled by the user either at the end or start of each scene, so no logic should be expected to handle transitions by this commandline feature.

**Comment by zhujisheng** [comment]
A scene is often quite long, especially if the movie lasts for several minutes. Typically, it needs to be split into multiple scenes for editing.
It's need to render the scenes in a specific order, while also providing a method to load objects displayed in the foreground of the previous scene into the next scene.
